metadata-ingestion/src/datahub/ingestion/source/affirm_artifact.py

```diff
@@ -0,0 +1,143 @@
+import sys
+import os
+import logging
+from dataclasses import dataclass, field
+from typing import Iterable, Sequence
+from ruamel.yaml import YAML
```

Comment:
How would this dependency be made available to the python environment where the `datahub` CLI would be invoked for the ingestion?

cc: @nickwu241

Reply:
good catch, it should be added here I think: https://github.com/Affirm/datahub/blob/v0.8.33.affirm/metadata-ingestion/setup.py#L276

Reply:
exactly @imtaos is correct